sormas-ui/src/main/java/de/symeda/sormas/ui/caze/AbstractCaseView.java

```diff
@@ -137,6 +138,12 @@ public void refreshMenu(SubMenu menu, String params) {
 
 		menu.removeAllViews();
 		menu.addView(CasesView.VIEW_NAME, I18nProperties.getPrefixCaption(CaseDataDto.I18N_PREFIX, Captions.caseCasesList));
+
+		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.LAB_MESSAGES)
```

Comment:
Would it be possible to only show this navigation option if the case originates from a lab message and to now show it for every case?

Reply:
It would be possible. Lab message are related to samples. One would therefore need to retrieve all samples for the case, and then check whether at least one lab message in the database relates to one of those samples. 

Do you think that this effort and additional system load is worth it? I'm not sure...

Reply:
Probably the check can just be a database query and we don't need to retrieve the samples from the db, just a yes or a no if the case can be joined with a sample and a lab message.

Since it will be ran for just a case I don't think it will put much pressure on the system and would probably be more helpful for the users then, as the number of cases created from lab messages probably won't outmatch those which are created directly.

So from my point of view it would make sense

Reply:
Okay. Wouldn't it be for the contact and event participant view as well?

Reply:
Yes, would be needed for those as well.